README.md

```diff
@@ -1,35 +1,92 @@
+
+You can also execute `kubectl` commands on K8s cluster via BotKube which helps debugging an application or cluster.
 
-![](botkube-title.jpg)
+<p align="center">
+<img src="./docs/assets/main-demo.gif" />
+</p>
 
 ## Getting started
 
 Please follow [this](https://botkube.io/installation/) for a complete BotKube installation guide.
 
-## Architecture
-![](/botkube_arch.jpg)
-- **Informer Controller:** Registers informers to kube-apiserver to watch events on the configured k8s resources. It forwards the incoming k8s event to the Event Manager.
-- **Event Manager:** Extracts required fields from k8s event object and creates a new BotKube event struct. It passes BotKube event struct to the Filter Engine.
-- **Filter Engine:** Takes the k8s object and BotKube event struct and runs Filters on them. Each filter runs some validations on the k8s object and modifies the messages in the BotKube event struct if required.
-- **Event Notifier:** Finally, notifier sends BotKube event over the configured communication channel.
-- **Bot Interface:** Bot interface takes care of authenticating and managing connections with communication mediums like Slack, Mattermost, Microsoft Teams and reads/sends messages from/to them.
-- **Executor:** Executes BotKube or kubectl command and sends back the result to the Bot interface.
+## Documentation
+
+For complete documentation visit [botkube.io](https://botkube.io).
+
+The documentation sources reside on the [botkube-docs](https://github.com/kubeshop/botkube-docs) repository under **content** directory.
+
+## Features
+
+<img src="./docs/assets/icons/terminal-box-line.svg" width="12%" align="right"/>
+
+### Execute `kubectl` commands
+
+Same old Kubectl syntax - just a new interface. You do not have to learn anything new! Plus you can configure which `kubectl` commands BotKube can execute. See [configuration](https://botkube.io/configuration/resource/) for details.
+
+<br /><br />
+
+<img src="./docs/assets/icons/question-answer-line.svg" width="10%" align="left"/>
+
+### Support multiple communicator
+
+BotKube integrates with Slack, Discord, Mattermost, Microsoft Teams, ElasticSearch and outgoing webhook. See [configuration](https://botkube.io/configuration/communication/) syntax for details.
+
+<br /><br />
+
+<img src="./docs/assets/icons/stack-line.svg" width="13%" align="right"/>
+
+### Supports Custom Resources
+
+BotKube can monitor literally any Kubernetes resource including Custom Resource. This enables you to configure alerts on events like - certificate issue if you use [cert-manager](https://cert-manager.io/) or backup failure in case you use backup tools like Velero or Kanister.
```

Comment:
```suggestion
BotKube supports literally any Kubernetes resource, including Custom Resources. For example, if you use [`cert-manager`](https://cert-manager.io/), you can get alerted about certificate issue, or backup failure in case you use backup tools like [Velero](https://velero.io/) or [Kanister](https://kanister.io/).
```